cpp/streamer/impl/batch/batch.cc

```diff
@@ -139,24 +157,34 @@ void Batch::read(const Config & config)
         finished_until(file_offset, common::ResponseCode::Success);
     }
 
-    if (file_offset < range.end)
+    if (file_offset < range.end && !stopped)
     {
         _reader->read(range.end - file_offset, buffer);
         finished_until(range.end, common::ResponseCode::Success);
     }
 
-    LOG(DEBUG) << "Finished reading successfuly from file " << path;
+    LOG(DEBUG) << "Finished reading " << i << "/" << num_chunks << " chunks of size " << config.fs_block_bytesize << " from file " << path << (stopped ? " - terminated" : " successfully");
```

Comment:
If the buffer is 8MB and we read it in 2*4MB requests we will do it in 2 operations and the log will show `i/num_chunks` which is `2/2`

If the buffer is 9MB we will read it in 2*4MB request + 1 request for the last, but we will still print `2/2`. Are we fine with it?

Reply:
Removed chunk size from the debug message